src/__tests__/ceramic_cas.ts

```diff
@@ -67,4 +67,118 @@ describe('Ceramic<->CAS integration', () => {
         expect(doc.content).toEqual(newContent)
         expect(doc.state.log.length).toEqual(4)
     })
+
+    test("multiple documents, multiple updates per anchor batch", async () => {
+        const content0 = { state: 0 }
+        const content1 = { state: 1 }
+        const content2 = { state: 2 }
+        const content3 = { state: 3 }
+        const content4 = { state: 4 }
+
+        // Test document creation
```

Comment:
So maybe it is the comment that is confusing me here..

We aren't really testing `createDocument` itself but we are testing that CAS responds correctly to `createDocument`. Is that accurate?

Reply:
well, on this line we're just creating documents, on line 90 below we begin testing that the creation is anchored properly.  I reworded this slightly to remove the emphasis on testing, is that clearer?

Reply:
Yes 👍